packages/web/src/scss/components/Dropdown/_theme.scss

```diff
@@ -0,0 +1,6 @@
+@use '@tokens' as tokens;
+
+$padding: tokens.$space-600;
+$border-radius: tokens.$radius-200;
+$shadow: tokens.$shadow-100;
+$background: tokens.$background-basic;
```

Comment:
can we sort this the same way as in the class directly?

1. padding
2. background
3. radius
4. shadow